torch/_decomp/decompositions.py

```diff
@@ -3512,7 +3512,7 @@ def multi_margin_loss_backward(
     z = margin - u + input
     z = z.clamp_min(0)
     g = 1.0 / (nframe * dim) if reduction == Reduction.MEAN.value else 1.0 / dim
-    g = torch.tensor(g, dtype=input.dtype, device=input.device)
+    g = g * torch.ones((), dtype=input.dtype, device=input.device)
```

Comment:
Had to replace `torch.tensor` with `torch.ones` and mul:

```diff
-    g = torch.tensor(g, dtype=input.dtype, device=input.device)
+    g = g * torch.ones((), dtype=input.dtype, device=input.device)
```

The former causes a heap overflow on the ASAN builder when running:

```py
pytest -rfxs test/functorch/test_aotdispatch.py -k test_aot_autograd_symbolic_exhaustive_nn_functional_multi_margin_loss_cpu_float32 -v --capture=no
```

Reply:
What about `torch.full`? But again, it'd be even better if we try autogenerating the backward.